docs/zh-CN/docs/data-table/index/prefix-index.md

```diff
@@ -41,6 +41,23 @@ under the License.
 
 而前缀索引，即在排序的基础上，实现的一种根据给定前缀列，快速查询数据的索引方式。
 
+需要注意的是为了防止索引膨胀，**在36字节限制和varchar截断限制的基础上，增加了列数限制（默认：3列）**，默认为**前三个KEY列**参与索引的构建，如果用户期望将更多的key列纳入到前缀索引中，需要在建表时修改PROPERTIES如下：
+
+```sql
+CREATE TABLE xxx
+(
+    ...
+)
+...
+PROPERTIES(
+    "shortkey_max_column_count" = "n" -- 其中 n 为用户设定的参与前缀索引构建的列数
```

Comment:
why do we need this limit? I think if user could set column count here, he could just set it to 100 and then he or she could use as many as columns in his alter sql.

Reply:
From the last discussion in the wechat group, the conclusion is that the default value of this parameter is to prevent the short index from being too large if so the index filtering does not work well. But the default value  is goot at all scenarios, so we make this parameter configurable. 

The key point is that there is a difference between the description in the document and the implementation in the engine, which will cause confusion to users